plugins/rhs_plugin/index.ts

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {connect} from 'react-redux';
+
+import {GlobalState} from 'types/store';
+import {PluginComponent} from 'types/store/plugins';
+
+import {getInt} from 'mattermost-redux/selectors/entities/preferences';
+
+import {getWorkTemplatesLinkedProducts} from 'mattermost-redux/selectors/entities/general';
+
+import {getCurrentUserId} from 'mattermost-redux/selectors/entities/users';
+
+import {getActiveRhsComponent, getPluggableId} from 'selectors/rhs';
+
+import {TutorialTourName, WorkTemplateTourSteps} from 'components/tours';
+
+import {suitePluginIds} from 'utils/constants';
+
+import RHSPlugin from './rhs_plugin';
+
+function mapStateToProps(state: GlobalState) {
+    const rhsPlugins: PluginComponent[] = state.plugins.components.RightHandSidebarComponent;
+    const pluggableId = getPluggableId(state);
+    const pluginComponent = rhsPlugins.find((element: PluginComponent) => element.id === pluggableId);
+    const pluginTitle = pluginComponent ? pluginComponent.title : '';
+
+    // work templates related code to show the tourtip
+    const activeRhsComponent = getActiveRhsComponent(state);
+    const pluginId = activeRhsComponent?.pluginId || '';
+
+    const currentUserId = getCurrentUserId(state);
+    const enableTutorial = state.entities.general.config.EnableTutorial === 'true';
```

Comment:
I believe this setting is for tutorials related to the onboarding but in the work templates the tooltips are more of a response of the action of creating the work templates. Not sure if we should not show if based on this config. Thoughts @esethna?

Reply:
not sure if could somehow eventually affect e2e, that's why I rather to keep it under that config value.

Reply:
+1 to Pablo, let's hide any tips or tutorials behind this flag for e2e purposes